enhancements/20220525-system-logging.md

```diff
@@ -0,0 +1,65 @@
+# System Logging
+
+## Summary
+
+We need to be able to support exporting harvester system logs outside the cluster, to allow for collecting and analyzing those logs to aid in debugging and harvester management. 
+
+### Related Issues
+
+- https://github.com/harvester/harvester/issues/577
+
+## Motivation
+
+### Goals
+
+List the specific goals of the enhancement. How will we know that this has succeeded?
+
+- The user can aggregate harvester logs in a centralized location
+- The user can view the aggregated harvester logs outside the cluster
+
+### Non-goals
+
+- this enhancement does not cover integration into the harvester / rancher UI (but ideally this will eventually be implemented as well)
+
+## Proposal
+
+We will deploy a new `ManagedChart` in the [Harvester Installer](https://github.com/harvester/harvester-installer) to
+install. The `ManagedChart` will deploy a `ClusterFlow` to select and aggregate interesting logs, which can be managed
```

Comment:
Please add definition or reference of `ClusterFlow`, `ClusterOutput`.

Reply:
Added reference links to `ClusterFloat` and `ClusterOutput` from Banzai docs\